wps-cloudformation-template.yaml

```diff
@@ -440,14 +447,15 @@ Resources:
     DeletionPolicy: Delete
     Properties:
       Type: MANAGED
-      ComputeEnvironmentName: !Sub 'JavaDuckSpotComputeEnvironment1-${AWS::StackName}'
+      ComputeEnvironmentName: !Sub 'JavaDuckSpotComputeEnvironment1-${customAmiId}-${AWS::StackName}'
```

Comment:
if a custom ami isn't passed in then this will end up as "JavaDuckSpotComputeEnvironment1--${AWS::StackName}", ie two dashes instead of one

Reply:
I'm OK with that.  The point here is to changed the environment name when the ami is changed to force cloudformation to delete the resource and recreate it.  I don't think extra logic to remove the two dashes is warranted.